src/nodes/NodeManager.ts

```diff
@@ -1044,71 +1040,138 @@ class NodeManager {
     pingTimeout: number | undefined,
     @context ctx: ContextTimed,
   ): Promise<void> {
-    this.logger.info('Syncing nodeGraph');
-    for (const seedNodeId of this.nodeConnectionManager.getSeedNodes()) {
-      // Check if the connection is viable
-      if (
-        (await this.pingNode(seedNodeId, undefined, {
-          timer: new Timer({
-            delay: pingTimeout ?? this.nodeConnectionManager.pingTimeout,
-          }),
-          signal: ctx.signal,
-        })) === false
-      ) {
-        continue;
-      }
+    const logger = this.logger.getChild('syncNodeGraph');
+    logger.info('Syncing nodeGraph');
+    // Getting the seed node connection information
+    const seedNodes = this.nodeConnectionManager.getSeedNodes();
+    // FIXME: remove or uncomment, need to decide
```

Comment:
What are all these comments for? Can they be turned into a new issue?

Reply:
That was part of the `syncNodeGraph` doing strategy 2. It was disabled while testing but I've re-enabled it now.